perf/jmh/playlist.xml

```diff
@@ -0,0 +1,19 @@
+<playlist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../TKG/playlist.xsd">
+	<include>testenv.mk</include>
+	<test>
+		<testCaseName>jmh-WriteBarrier</testCaseName>

```

Comment:
Is this going to be the name of the tests as shown in the UI for the user? If so, won't it be better to mention that this is going to run all tests in JMH_JDK micros?